setup.py

```diff
@@ -109,6 +109,7 @@
             'paasta_setup_chronos_job=paasta_tools.setup_chronos_job:main',
             'paasta_chronos_rerun=paasta_tools.chronos_rerun:main',
             'paasta_cleanup_maintenance=paasta_tools.cleanup_maintenance:main',
+            'docker_hostname_wrapper=paasta_tools.docker_hostname_wrapper:main',
```

Comment:
Sorry for so much review, but long term we would like to use this wrapper for more things other than hostname.
also I prefer that new script have the prefix `paasta_` in them so that they can't possibly be mistaken for other tools on the filesystem.

Can you call it just `paasta_docker_wrapper` ?